staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go

```diff
@@ -279,7 +279,7 @@ func Run(g *Generator) {
 		cmd := exec.Command("protoc", append(args, path)...)
 		out, err := cmd.CombinedOutput()
 		if len(out) > 0 {
-			log.Printf(string(out))
+			log.Printf("%s", string(out))
```

Comment:
Better to just do `log.Print(string(out))`. Same below.

Reply:
Why `log.Printf(string(out))` here? Is there any reason?

Reply:
Could you please share your points about how to dealing with the static check error? Ignore this line?

Reply:
@RainbowMango Just use `log.Print` instead of `log.Printf` !

Reply:
Oh....

Reply:
Done. Thanks @SataQiu @tallclair